src/ngx_http_lua_util.h

```diff
@@ -9,6 +9,12 @@
 #define _NGX_HTTP_LUA_UTIL_H_INCLUDED_
 
 
+#ifndef DDEBUG
+#define DDEBUG 0
+#endif
+#include "ddebug.h"
```

Comment:
The thing is that we never include this snippet in our header files. It should only be included in individual compilation units since we need the ability to turn DDEBUG on and off in individial compilation units.

Reply:
@agentzh 
However, `dd` macro is used in `ngx_http_lua_util.h`:
https://github.com/openresty/lua-nginx-module/blob/master/src/ngx_http_lua_util.h#L328

If we don't add the definition, those usages are no op.

Reply:
@spacewander Let's remove those maybe? It seems like they would be awfully noisy anyways, plust they could be invoked from the caller's POV instead (e.g., callers of `ngx_http_lua_get_lua_vm()`, wherever the developer is currently debugging).

Reply:
That `dd` line should be guarded by `#ifdef DDEBUG`/`#endif`.

Reply:
That's a good solution too :)

Reply:
I submit the change in another pull request： https://github.com/openresty/lua-nginx-module/pull/1418